proximo-server/metrics.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// Counters struct holds various counters needed for app metrics.
+type counters struct {
+	ErrorCounter           prometheus.Counter
+	SinkMessagesCounter    *prometheus.CounterVec
+	SourcedMessagesCounter *prometheus.CounterVec
+}
+
+// NewCounters returns metric counters needed for the app.
+func NewCounters() counters {
+	c := counters{
+		ErrorCounter: prometheus.NewCounter(prometheus.CounterOpts{
+			Name: "errors_total",
+			Help: "A counter of the number of errors",
+		}),
+		SinkMessagesCounter: prometheus.NewCounterVec(
```

Comment:
These counters are handled by go-pubsub automatically, please remove `SinkMessagesCounter` and `SourcedMessagesCounter`. 

See how they should be defined:
* kafka sink: https://github.com/utilitywarehouse/opsgenie-alert-producer/blob/master/main.go#L62-L69
* kafka source: https://github.com/utilitywarehouse/diff-to-kafka/blob/20b3e45608831c4b524bd6ec72150bd08ecd4128/main.go#L64-L71

Reply:
@ggirtsou I think that doesn't apply to this project as it's not using go-pubsub.